saw-remote-api/python/saw/__init__.py

```diff
@@ -343,7 +343,7 @@ def view(v: View) -> None:
 
 
 def cryptol_load_file(filename: str) -> None:
-    __get_designated_connection().cryptol_load_file(filename)
+    __get_designated_connection().cryptol_load_file(filename).result()
```

Comment:
A general question I have: how do you know when to call `result()` and when not to? I noticed that some function, such as `cryptol_load_file` and `llvm_load_module`, invoke `result()` on their respective `Commands` while other functions, such as `llvm_verify`, do not.

Reply:
In the body of `llvm_verify` there are no calls to `result` but there are calls to the `stdout` and `stderr` methods which will perform the same "wait for a response" action that `result` does.

I personally want to change this slightly and call `result` instead to get the `yay` or `nay` response from the server (see https://github.com/GaloisInc/saw-script/issues/1192), but that's for another PR.